proto/viam/common/v1/common.proto

```diff
@@ -131,6 +131,7 @@ message Transform {
   string reference_frame = 1;
   // the pose of the above reference frame with respect to a different observer reference frame
   PoseInFrame pose_in_observer_frame = 2;
+  Geometry geometry = 3;
```

Comment:
The scope says this will be optional.

Reply:
Sorry; hadn't pushed that update from my local yet, it's updated now.